### PR TITLE
agent-tracking-docai-processor

### PR DESCRIPTION
**Pull request by anuradha-bajpai-google**
Added label for agent tracking docai processor

### CODE REVIEW

**Review by dharmez — DISMISSED** [review]

File: components/processing/form_parser/deployment/main.tf [hunk 1]
```diff
@@ -12,6 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+locals {
```
Comment (dharmez) [inline components/processing/form_parser/deployment/main.tf:15]: This labels are applied for Revenue Tracking.

**Comment by dharmez** [comment]
The Agent based tracking tags are applied in the python code. The Revenue tracking labels are applied in terraform. The Agent based tracking tags are missing for the Form processor.

File: components/processing/form_parser/src/main.py [hunk 2]
```diff
@@ -52,10 +53,12 @@ def batch_process_documents(
         gcs_output_uri: GCS directory to store the out json files,
         gcs_input_prefix: GCS directory to store input files to be processed
     """
+  USER_AGENT = "cloud-solutions/eks-agent-builder-v1"
```
Comment (dharmez) [inline components/processing/form_parser/src/main.py:56]: Please use the following value for the USER_AGENT string:
"cloud-solutions/eks-docai"

File: components/processing/form_parser/src/load_data_in_bigquery.py [hunk 3]
```diff
@@ -28,8 +29,10 @@ def load_rows_into_bigquery(rows_to_insert, table_id):
         rows_to_insert: list of rows with each row as json string,
         table_id: bigquery table id
     """
+  USER_AGENT = "cloud-solutions/eks-agent-builder-v1"
```
Comment (dharmez) [inline components/processing/form_parser/src/load_data_in_bigquery.py:32]: Please use the following value for the USER_AGENT string:
"cloud-solutions/eks-docai"

Comment (anuradha-bajpai-google) [reply components/processing/form_parser/src/load_data_in_bigquery.py]: updated

File: components/processing/form_parser/src/main.py (hunk 2 above)
Comment (anuradha-bajpai-google) [reply components/processing/form_parser/src/main.py]: updated

File: components/processing/form_parser/deployment/main.tf (hunk 1 above)
Comment (anuradha-bajpai-google) [reply components/processing/form_parser/deployment/main.tf:15]: updated

**Review by dharmez — APPROVED** [review]
Both Agent based tracking tags and revenue tracking labels were applied in terraform and code.

**Pull request merged**